wasm/jsapi/wasm-module-builder.js

```diff
@@ -553,9 +553,9 @@ class WasmModuleBuilder {
     return this.num_imported_funcs++;
   }
 
-  addImportedGlobal(module, name, type) {
+  addImportedGlobal(module, name, type, mutable = false) {
     let o = {module: module, name: name, kind: kExternalGlobal, type: type,
-             mutable: false}
+             mutable: mutable};
```

Comment:
```suggestion
             mutable};
```

Reply:
I'm going to not do this for consistency with the rest of the file and the V8 copy.